specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -3064,6 +3071,31 @@
       },
       "discriminator": "jobType"
     },
+    "BMSAADPropertiesQueryObject": {
```

Comment:
Why do we have to add this model if this is not used anywhere?

Reply:
We are adding this model as x-ms Odata filter in AadProperties_Get API